manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This is cool 🚀.

Though I noticed this includes only x192 and x512 android icon sizes (these are new, cool 🚀) do include the rest of the icon sizes available in this -> **images/favicon.ico/manifest.json** also the density attributes.
NB: Consistency is key in JSON structure: src -> sizes -> type -> density

Reply:
I think there is no need to add the density attribute.
See here https://github.com/RealFaviconGenerator/realfavicongenerator/issues/228